src/algebra/field/opposite.lean

```diff
@@ -13,19 +14,36 @@ import algebra.ring.opposite
 > Any changes to this file require a corresponding PR to mathlib4.
 -/
 
-variables (α : Type*)
+variables {α : Type*}
```

Comment:
I'd prefer that we left this as is to minimize downstream disruption.

Reply:
I've reverted the revert because it messes up with `op_rat_cast`/`unop_rat_cast`.